google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudBreakpointHandler.java

```diff
@@ -341,7 +341,7 @@ public void registerBreakpoint(
       BreakpointManager.getJavaBreakpoint(xIdeBreakpoint);
     if (!(cloudIdeBreakpoint instanceof CloudLineBreakpointType.CloudLineBreakpoint)) {
       LOG.error("breakpoint was not of the correct type to create on the cloud.  It was not a " +
-                "CloudLineBreakpoint");
+                "CloudLineBreakpoint"); //TODO akerekes: use class.getName() instead of hard coded name
```

Comment:
I know, I like it, but still seems fragile depending on the tool to take care of it. Anyway, I'll remove the TODO, and submit a separate PR for the change, I don't want to mix it into this one.